ui/cypress/e2e/variables.test.ts

```diff
@@ -73,4 +73,28 @@ describe('Variables', () => {
 
     cy.getByTestID('resource-card').should('have.length', 1)
   })
+
+  it('can rename a variable', () => {
+    cy.get<Organization>('@org').then(({id}) => {
+      cy.createVariable(id)
+    })
```

Comment:
Probably want to put this in a `beforeEach` to guarantee this gets created before the test.